src/main/groovy/org/transmartproject/batch/highdim/mrna/data/MrnaDataJobConfiguration.groovy

```diff
@@ -59,10 +59,9 @@ class MrnaDataJobConfiguration extends AbstractStandardHighDimJobConfiguration {
                 partitionByColumn: 'trial_name',
                 partitionByColumnValue: studyId,
                 sequence: Sequences.MRNA_PARTITION_ID,
-                indexes: [
-                        ['assay_id'],
-                        ['probeset_id'],
-                        ['patient_id'],
+                primaryKey: ['assay_id', 'probeset_id'],
+                foreignKeys: [
+                        assay_id: [table: Tables.SUBJ_SAMPLE_MAP, column: 'assay_id', onDelete: 'CASCADE']
```

Comment:
We always query HD data tables by restricting both `assay_id` and `probeset_id` at the same time. Don't we? I would see the need to have separate index for `probeset_id` in case we would retrieve the data by `probeset_id` only somewhere in the code. Does it make sense for you?